frame/aura/src/lib.rs

```diff
@@ -79,7 +80,22 @@ pub mod pallet {
 	pub struct Pallet<T>(sp_std::marker::PhantomData<T>);
 
 	#[pallet::hooks]
-	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {}
+	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
+		fn on_initialize(_: T::BlockNumber) -> Weight {
+			if let Some(new_slot) = Self::current_slot_from_digests() {
```

Comment:
When you call into the runtime for doing some random runtime call, this will not be provided. So, we can not `expect` it here. (BABE is doing the same btw)